src/common/berthDetails/BerthDetailsContainer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useTranslation } from 'react-i18next';
+
+import BerthDetails from './BerthDetails';
+import LoadingSpinner from '../spinner/LoadingSpinner';
+import { BERTH_DETAILS_QUERY } from './queries';
```

Comment:
This uses a query, so it should probably be in `features`

Reply:
`this` being `common/berthDetails`

Reply:
Moved `berthDetails` to `features` dir